52.md

```diff
@@ -0,0 +1,142 @@
+
+NIP-52
+======
+
+Issued Claims
+-------------
+
+`draft` `optional` `author:neilck`
+
+This NIP is currently in draft, and review / comments are sought. May overlap with [NIP-32](https://github.com/nostr-protocol/nips/blob/39bfb2db32899dd2eb4a7c9a313f27103a18409b/32.md).
+
+This NIP defines new event kinds for the issuing of claims. 
+
+It reserves two event kinds (30001, 1001) for immediate use.
+
+- `30001 - issue claim`
+- `1001 - refresh claim request`
+
+An issue claim event records what one user attests is true about another user. 
+
+## Kind 30001: Issue Claim
+
+Sent by a user acting as an issuer, submitting a claim about another user, the claim's subject.
+
+```json
+{
+  "id": <32-bytes lowercase hex-encoded sha256 of the the serialized event data>
+  "pubkey": <32-bytes lowercase hex-encoded public key of the issuer>,
+  "created_at": <unix timestamp in seconds>,
+  "kind": 30001,
+  "tags": [
+    ["p", <32-bytes hex of the subject's pubkey>, <recommended relay URL>],
+    ["d", <32-bytes hex of the subject's pubkey + claim type>],
+    ["claim_type", <optional string specifying claim type>],
+    ["is_maintained", <optional boolean, yes if refresh claim supported>]
+  ],
+  "content": <claim data>,
+  "sig": <64-bytes signature of the sha256 hash of the serialized event data, which is the same as the "id" field>
+}
+```
+
+`"d" tag` is required as the event is treated as a parameterized replaceable event [NIP-33](https://github.com/nostr-protocol/nips/blob/master/33.md). Issuers issuing more than one type of claim will need to append a claim type identifier to the subject's pubkey hex to prevent overwrites.
+
+The p tag MUST contain exactly one entry, the pubkey of the claim's subject.
+
+Self-issued claims (issuer is the subject) is specifically allowed.
+
+
+## Kind 1001: Refresh Claim Request
+
+A claim may change over time. This event allows a user to request the issuer submit an updated issue claim event for the same subject and claim_type.
+
+```json
+{
+  "id": <32-bytes lowercase hex-encoded sha256 of the the serialized event data>
+  "pubkey": <32-bytes lowercase hex-encoded public key of the sender>,
+  "created_at": <unix timestamp in seconds>,
+  "kind": 1001,
+  "tags": [
+        ["e", <original issue claim event-id>, <relay-url>]
```

Comment:
I might not understand well what's going on here but to reference a replaceable event it's best to reference it by it's identifier and not the event ID - in this case `<pubkey>:<d-tag>` or  `<kind>:<pubkey>:<d-tag>` to not lose the referenced event.

Reply:
Document updated.

Reply:
event removed with NIP focused on Badges vs Issued Claims in general